src/plugins/intel_gpu/src/graph/impls/ocl/bucketize.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "bucketize_inst.h"
+#include "primitive_base.hpp"
+#include "impls/implementation_map.hpp"
+#include "kernel_selector_helper.h"
+#include "bucketize/bucketize_kernel_selector.h"
+#include "bucketize/bucketize_kernel_ref.h"
+#include "intel_gpu/runtime/error_handler.hpp"
+
+using namespace cldnn;
+
+namespace cldnn {
+namespace ocl {
+struct bucketize_impl : typed_primitive_impl_ocl<bucketize> {
+    using parent = typed_primitive_impl_ocl<bucketize>;
+    using parent::parent;
+
+    std::unique_ptr<primitive_impl> clone() const override {
+        return make_unique<bucketize_impl>(*this);
+    }
+
+public:
+    static primitive_impl* create(const bucketize_node& arg) {
+        auto bucketize_params = get_default_params<kernel_selector::bucketize_params>(arg);
+        auto bucketize_optional_params =
+                get_default_optional_params<kernel_selector::bucketize_optional_params>(arg.get_program());
```

Comment:
Please try to use 4-spaces tab instead of this one. Also if line length is less or equal than 160 symbols you can stay it without splitting on 2 separate lines.